--- yahtzee_sbi-1.04-orig/Makefile	1999-12-04 15:38:55.000000000 -0600
+++ yahtzee_sbi-1.04/Makefile	2006-05-07 19:55:02.058750000 -0500
@@ -6,10 +6,10 @@
 LIBS = -lncurses -ltermcap # For Linux
 #LIBS = -lcurses -ltermcap # For Solaris
 
-BIN = yahtzee
+BIN = yahtzee.exe
 
-CFLAGS = -Wall -g -I/usr/include/ncurses -DFRENCH
-#CFLAGS = -Wall -g -I/usr/include/ncurses -DENGLISH
+#CFLAGS = -Wall -g -I/usr/include/ncurses -DFRENCH
+CFLAGS = -O2 -pipe -Wall -g -I/usr/include/ncurses -DENGLISH
 
 $(BIN): $(OBJ) lang.h english.h french.h
 	$(CC) -o $@ $(OBJ) $(LIBS)
